test/shared/fixtures.ts

```diff
@@ -346,9 +346,15 @@ export const airdropCampaignDataFixture: Fixture<AirdropCampaignDataFixtureType>
 
   const airdropCampaignDataFactory = await ethers.getContractFactory("AirdropCampaignData");
 
-  const airdropCampaignData = (await airdropCampaignDataFactory
+  // --- Old way of deployign the contract
+  /* const airdropCampaignData = (await airdropCampaignDataFactory
     .connect(deployer)
-    .deploy(process.env.BACKEND_WALLET_ADDRESS)) as AirdropCampaignData;
+    .deploy(process.env.BACKEND_WALLET_ADDRESS)) as AirdropCampaignData; */
+
+  // --- New way of deployign the contract - UPGRADABLE
+  const airdropCampaignData = (await upgrades.deployProxy(airdropCampaignDataFactory, [
+    process.env.BACKEND_WALLET_ADDRESS,
```

Comment:
Inheirt UUPS in smart contract and deploy this way
await upgrades.deployProxy(airdropCampaignDataFactory, [process.env.BACKEND_WALLET_ADDRESS], {
    initializer: "initialize",
    kind: "uups"
  });